envoy_carrier.proto

```diff
@@ -1,7 +1,23 @@
+// EnvoyCarrier defined here is nearly identical to but NOT wire-compatible
+// with TracerState defined here:
+// https://github.com/opentracing/basictracer-go/blob/master/wire/wire.proto
+
 syntax = "proto3";
 
-package lightstep.envoy;
```

Comment:
Ah. Didn't mean to remove this, but I'm not sure we should keep the "lightstep" there, since this is meant to be an Envoy thing that other Tracers could implement.